exporter/stackdriverexporter/stackdriver.go

```diff
@@ -195,3 +202,11 @@ func (te *traceExporter) pushTraces(ctx context.Context, td pdata.Traces) (int,
 
 	return numSpans - goodSpans, componenterror.CombineErrors(errs)
 }
+
+func getTimeout(cfg *Config) time.Duration {
```

Comment:
Can we just set the default value in [createDefaultConfig ](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/exporter/stackdriverexporter/factory.go#L50) instead? I can't remember if viper overrides this value if not set but I'm pretty sure it should work.

Reply:
Yes please set it as default in the default config. if user overwrites that should have priority.

Reply:
Done.